conda_build/index.py

```diff
@@ -934,6 +936,13 @@ def index(self, patch_generator, hotfix_source_repo=None, verbose=False, progres
                 self._write_channeldata_rss(channel_data, package_mtimes, hotfix_source_repo)
                 self._write_channeldata(channel_data)
 
+                # Step 8. Sign the repodata
+                if gpg_wrapper:
+                    for subdir in subdirs:
+                        for _file in ["repodata.json.bz2", "repodata.json", "repodata2.json"]:
```

Comment:
I couldn't find any easy way to iterate over all the repodata files. Hence the hardcoding.